corehq/apps/data_interfaces/forms.py

```diff
@@ -394,15 +403,17 @@ def __init__(self, domain, *args, **kwargs):
             ),
         )
 
-        self.case_type_helper = HQFormHelper()
-        self.case_type_helper.form_tag = False
-        self.case_type_helper.layout = Layout(
+        self.form_beginning_helper = HQFormHelper()
+        self.form_beginning_helper.form_tag = False
+        self.form_beginning_helper.layout = Layout(
             Fieldset(
                 _("Rule Criteria"),
-                Field('case_type', data_bind="value: caseType", css_class="hqwebapp-select2")
+                Field('case_type', data_bind="value: caseType", css_class="hqwebapp-select2"),
+                Field('criteria_operator', data_bind="value: criteriaOperator", css_class="hqwebapp-select2")
```

Comment:
There's a "select toggle" widget we often use for choices that only have two options: https://www.commcarehq.org/styleguide/molecules/#molecules-selections

It might be good here (or it might be terrible because of how long the option text is).

Reply:
<img width="811" alt="Screen Shot 2022-04-27 at 2 50 14 PM" src="https://user-images.githubusercontent.com/6729291/165598422-42f19cfe-5d8b-4c19-aa89-d5b543ac904d.png">

what do you think?

Reply:
I like it!